themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/linked-accounts-page/linked-accounts-page.component.html

```diff
@@ -1 +1,126 @@
-<h1>Placeholder for Linked Accounts Page (formerly known as Federated Identity)</h1>
\ No newline at end of file
+<!--content-->
+<div class="page-header">
+  <h1>Linked Accounts</h1>
+</div>
+<div class="col-sm-12 card-pf card-linked-account">
+  <div class="card-pf-body row">
+    <div class="col-md-3">
+      <div class="card-pf-subtitle">
+        Authorized Provider
+      </div>
+      <div class="introMessage">
+        <p>Authorized Providers linked with your account.</p>
+      </div>
+    </div>
+    <div class="col-md-9 linked-account-list">
```

Comment:
Do we need to add the second new css "linked-account-list", as above I found you already added "card-linked-account"?